librz/core/linux_heap_glibc.c

```diff
@@ -1358,38 +1357,63 @@ void GH(print_malloc_info)(RzCore *core, GHT m_state, GHT malloc_state) {
 	}
 }
 
+char *GH(rz_bin_num_to_type)(int bin_num) {
+	if (bin_num == 0) {
+		return rz_str_new("Unsorted");
+	} else if (bin_num >= 1 && bin_num <= NSMALLBINS - 1) {
+		return rz_str_new("Small");
+	} else if (bin_num >= NSMALLBINS && bin_num <= NBINS - 2) {
+		return rz_str_new("Large");
+	}
+	return NULL;
+}
+
+void GH(rz_heap_bin_free)(RzHeapBin *bin) {
+	free(bin->type);
+	free(bin->message);
+	rz_list_free(bin->chunks);
+	free(bin);
+}
 /**
  * \brief Get list of chunks of <bin_num> bin from NBINS array of an arena.
  * \param core RzCore pointer
  * \param main_arena MallocState struct of arena
  * \param bin_num bin number of bin whose chunk list you want
  * \return RzList of RzHeapChunkListItem for the bin
```

Comment:
Since you changed the return type please update also the documentation of the function